tests/test_number_spelling.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""Testing the spelling of numbers
+
+:copyright: (c) 2013 by Szabolcs Blága
+:license: BSD, see LICENSE in babel for more details.
+"""
+from __future__ import unicode_literals
+
+import pytest
+
+from babel import numbers
+
+def test_public_interface():
```

Comment:
Can you add the test implementation of this? Also, `test_context_framework` lacks implementation too.